src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -241,25 +211,17 @@ class Preview extends React.Component<Props> {
     };
 
     @action handlePreviewWindowClick = () => {
-        const {previewStore} = this;
-
-        if (!previewStore) {
-            throw new Error('The preview cannot be updated if the "PreviewStore has not been initialized yet."');
-        }
-
-        this.previewWindow = window.open(previewStore.renderRoute);
+        this.previewWindow = window.open(this.previewStore.renderRoute);
         this.previewWindow.addEventListener('beforeunload', action(() => {
             this.previewWindow = undefined;
         }));
     };
 
     render() {
-        const {previewStore} = this;
```

Comment:
Was there any specific reason to remove the destructuring or just because of consistence to the other used vars?

Reply:
It was about consistency :slightly_smiling_face: